modules/custom/d_p_side_by_side/templates/paragraph--d-p-side-by-side.html.twig

```diff
@@ -49,23 +49,27 @@
   <section {{ wrapper_attributes }}>
     <div{{ attributes.addClass(classes) }}>
       {% block tiles_content %}
-        <div class="d-p-tiles-wrapper expandable-content">
-          <div class="container">
-            {{ content.field_d_media_icon }}
+        {% if (content.field_d_media_icon|render) or (content.field_d_main_title|render) or (content.field_d_long_text|render)  %}
+        <div class="container side-header">
+          {{ content.field_d_media_icon }}
+          {% if (content.field_d_main_title|render) or (content.field_d_long_text|render)  %}
             <div class="text-content">
               {{ content.field_d_main_title }}
               {{ content.field_d_long_text }}
             </div>
```

Comment:
ja to bym po prostu wyprintowal tutaj 
`{{ content|without('field_d_p_sbs_items', 'field_d_cta_link') }}`

bo:
`text-content` jest chyba potrzebne tylko do tego, zeby ustawic padding, a mozna go ustawic inaczej (jakies `.field + d-p-tiles-wrapper { padding }` czy cos)
![image](https://user-images.githubusercontent.com/37533838/88180702-0f91d980-cc2e-11ea-8132-3f1135576e99.png)

unikniemy wtedy tej ifologi i bedzie o wiele czysciej..